plugins.txt

```diff
@@ -10,7 +10,7 @@ azure-container-agents:253.vd2f5cd5c5040
 azure-credentials:254.v64da_8176c83a
 azure-sdk:132.v62b_48eb_6f32f
 azure-vm-agents:859.v7213476e4fea_
-basic-branch-build-strategies:71.vc1421f89888e
+basic-branch-build-strategies:81.v05e333931c7d
```

Comment:
https://github.com/jenkinsci/basic-branch-build-strategies-plugin/releases/tag/81.v05e333931c7d => no changes for us